### PR TITLE
Google Suggest extension

### PR DESCRIPTION
**Pull request by HansCz**
Let me know what you think :)

### CODE REVIEW

File: google_suggest/__init__.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+user_agent = "Mozilla/5.0"
+limit = 20
+
+
+def initialize():
+    global baseurl
+
+def handleQuery(query):
+    if query.isTriggered:
+
```
Comment (djmbritt) [inline google_suggest/__init__.py]: I would consider adding:
sleep(0.2)
if not query.isValid:
            return

That way avoid being rate-limited by google.
I've tried it with a couple of other google end points, and after a while they always end up limit the number of calls you can make to their API endpoint.

**Comment by djmbritt** [comment]
Except for the comment I made about constantly making calls to the URL, I really like this. Works great.

File: google_suggest/__init__.py (hunk 1 above)
Comment (HansCz) [reply google_suggest/__init__.py]: I wasn't aware that they rate-limited. Thanks for the heads up.
I'll put it in.
Will also go see if I can find specifics on how the rate limiting functions, so I can add a warning in a comment for anybody who wants to alter the code.

Comment (HansCz) [reply google_suggest/__init__.py]: I added your suggested change. 

Unfortunately I was unable to find any definite numbers on when and how the rang-limiting kicks in, so I left it out.

**Comment by HansCz** [comment]
Have you considered whether to merge this?

**Comment by djmbritt** [comment]
I'd like for this to be merged. But I don't have write access. 
I do know that the Manuel is busy at the moment and doesn't have time. 

The python-community repo is open though, for other community members to merge pull requests.

**Comment by ManuelSchneid3r** [comment]
Unfortunately I let you all wait pretty long. Now I have the problem that I don't know which PR's are valid at all. Therefore I will close all PR's for tentatively. If you want me to reopen the PR let me know. Also it would be nice if you could resolve potential conflicts due to changes in the repo. Sorry for the inconveniences, but I am working on it for a week now. I'm in a hurry. Note that this is a copy paste message. 😉

**Pull request closed without merging**